records_mover/db/bigquery/loader.py

```diff
@@ -82,21 +103,51 @@ def load(self,
              table: str,
              load_plan: RecordsLoadPlan,
              directory: RecordsDirectory) -> int:
+
+        if directory.scheme != 'gs':
```

Comment:
what are the options for `directory.scheme`?

Reply:
Right now it's the URL schemes that Records Mover supports reading from:
* `gs`
* `s3`
* `file`
* `http`

Reply:
Which ones are supported at a given time depends on what libraries and credentials you have:

https://github.com/bluelabsio/records-mover/blob/master/records_mover/url/resolver.py#L21-L50